rocAL/rocAL/include/meta_data/meta_data_graph.h

```diff
@@ -34,6 +34,7 @@ class MetaDataGraph
 public:
     virtual ~MetaDataGraph()= default;
     virtual void process(MetaDataBatch* meta_data) = 0;
+    virtual void update_meta_data(MetaDataBatch* meta_data, decoded_image_info decode_image_info) = 0;
```

Comment:
This function can be removed - refer https://github.com/SundarRajan28/MIVisionX/commit/11126d6bbf649bfbc69425f2888527f1195891a8 and https://github.com/SundarRajan28/MIVisionX/commit/a43f7ac11796870e9b4e1f01bd1a94404193ef5d